examples/run-workflow/main.go

```diff
@@ -96,6 +106,29 @@ ExitLoop:
 			break ExitLoop
 		case <-time.After(5 * time.Second):
 		}
+		if showEvents {
+
+			events, newNbEvents, err := client.EventService().GetEventsForApplicationEnvironment(ctx, envID, 0, nbEvents+100000)
```

Comment:
Why `from` can't help here to get only newer events?
Why always starting from `0`

Reply:
0 is the index of the latest event. A4C returns events sorted by date descending, so when using an index other than 0 we mistt the last events

Reply:
Ok misunderstood this sorry. 
But why  `nbEvents+100000`? Can't we do something like a more surgical strike?

Reply:
here this just just a basic example of usage, but yes I can do it